### PR TITLE
[Documentation][Book] Inventory

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | no
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/inventory.rst [hunk 1]
```diff
@@ -4,63 +4,42 @@
...
+OrderInventoryOperator
+----------------------
 
-InventoryOperator
------------------
+Inventory Operator is the service responsible for managing the stock amounts of every *ProductVariant* on an Order with the following methods:
 
-Inventory operator is the service responsible for managing the stock amounts of every *ProductVariant* with following methods:
+* ``hold`` - is called when the order's checkout is completed, it transfers the inventory units from onHand to onHold,
+* ``sell`` - is called when the order's payment are assigned with the state ``paid``. The inventory items are then removed from onHold.
```
Comment (tuka217) [inline docs/book/inventory.rst]: `and onHand`

File: docs/book/inventory.rst [hunk 2]
```diff
@@ -4,63 +4,42 @@
...
+* ``release`` - is a way of making onHold items of an order back onHand,
```
Comment (tuka217) [inline docs/book/inventory.rst]: `making items not reserved`?

File: docs/book/inventory.rst (hunk 1 above)
Comment (Kibsoon) [inline docs/book/inventory.rst]: sometimes you use ',' and sometimes '.' at the end of the line

**Comment by pjedrzejewski** [comment]
Thanks Magda!

**Pull request merged**